MiniAn/minian_main.py

```diff
@@ -365,6 +365,14 @@ def cnmf1(Y_hw_chk, intpath, A, C, C_chk, Y_fm_chk, chk, minian_parameters):
                             chunks={"unit_id": -1, "frame": chk["frame"]})
         sig = save_minian(sig_mrg.rename("sig_mrg"), intpath, overwrite=True)
 
+    # making unit_ids start with 1 instead of 0
```

Comment:
I'm not sure it is the right place to do that. Maybe it would be better to do it just before the call of `cnmf2`. Also I look in the doc but there are no option to force to set the first id to 1 and not 0 but I didn't look too much.

Furthermore you can remove the for loop and do something like that ` ids_temp1 = A["unit_id"].values +1` or `ids_temp1 = np.array(A["unit_id"].values) +1`